queue/call_preview.go

```
@@ -3,6 +3,7 @@ package queue
 import (
 	"encoding/json"
 	"fmt"
+
```

golangci-lint (2.12.2)

[error] File is not properly formatted

(gci)